gramine_ratls_verify_test.go

```diff
@@ -26,6 +28,15 @@ var isv_prod_id []byte
 var isv_svn []byte
 
 func setup() {
+	// Set DEBUG flag for logs
+	os.Setenv("DEBUG", "1")
+
+	// Set `RA_TLS_ALLOW_OUTDATED_TCB_INSECURE` environment variable to 1
```

Comment:
(because the unit tests are working with a potentially out of date attestation quote)